pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[tool.poetry]
+name = "to-do-app"
+version = "0.1.0"
+description = ""
+authors = ["Joanna Cheng <joanna.cheng@softwire.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+astroid = "2.4.1"
+certifi = "^2020.6.20"
+chardet = "^3.0.4"
+click = "^7.1.2"
+colorama = "^0.4.3"
+Flask = "^1.1.2"
+idna = "^2.10"
+isort = "^5.4.2"
+itsdangerous = "^1.1.0"
+Jinja2 = "^2.11.2"
+lazy-object-proxy = "1.4.3"
+MarkupSafe = "^1.1.1"
+mccabe = "^0.6.1"
+pylint = "^2.6.0"
+python-dotenv = "^0.14.0"
+requests = "^2.24.0"
+six = "^1.15.0"
+toml = "^0.10.1"
+urllib3 = "^1.25.10"
+Werkzeug = "^1.0.1"
+wrapt = "^1.12.1"
+pytest = "^6.0.1"
+selenium = "^3.141.0"
```

Comment:
You've included nested dependencies, which isn't necessary (you shouldn't need much more than python, flask, requests, python-dotenv, pytest, selenium)

Also, pytest and selenium belong in the dev-dependencies section (so that you can optionally skip them, e.g. on a production server)